ui/tsconfig.base.json

```diff
@@ -15,7 +15,8 @@
     "isolatedModules": true,
     "noUncheckedIndexedAccess": true,
     "declaration": true,
-    "declarationMap": true
+    "declarationMap": true,
+    "pretty": true
   },
   // For builds that use ts-node to compile the configs (e.g. webpack, jest)
```

Comment:
Unrelated to this PR, but are comments supported in .json? My editor shows linting errors

Reply:
Yeah, `tsconfig` is JSONC and supports them. Not all JSON though obviously. If you're using VS Code and it doesn't do it for you automatically, you can switch the file type down in the bottom right corner by clicking on it:

![Screen Shot 2022-09-26 at 2 12 52 PM](https://user-images.githubusercontent.com/428023/192370927-af8d382e-698b-4d2a-83bd-443fa17ec51d.png)

Reply:
Ah I see, the file I was noticing had the issue was actually turbo.json which VSCode doesn't automatically switch. Seems like[ they fixed it ](https://github.com/Microsoft/vscode/issues/50974) for tsconfig though